plugin/core/types.py

```diff
@@ -236,8 +236,11 @@ def r(name: str, default: Union[bool, int, str, list, dict]) -> None:
 
         set_debug_logging(self.log_debug)
 
-    def document_highlight_style_to_add_regions_flags(self) -> int:
-        return _settings_style_to_add_regions_flag(self.document_highlight_style)
+    def document_highlight_style_to_add_regions_flags(self) -> Tuple[int, int]:
```

Comment:
Nit. I would just name it `document_highlight_style_region_flags` since from the perspective of the caller, the fact that it maps the user setting to to regions shouldn't be relevant.